src/test/java/nl/tudelft/contextproject/model/entities/VRPlayerTest.java

```diff
@@ -182,6 +184,54 @@ public void testUpdateExploration_range() {
 	}
 	
 	/**
+	 * Tests if a bomb gets picked up if pickup is pressed.
+	 */
+	@Test
+	public void testPickUpBomb() {
+		TestUtil.mockGame();
```

Comment:
the game is already mocked before every test of this class (because of `setMockGame(true)` at line 38), so this is not needed.